src/PSR7Session/Http/SessionMiddleware.php

```diff
@@ -20,154 +20,21 @@
 
 namespace PSR7Session\Http;
 
-use Dflydev\FigCookies\FigResponseCookies;
-use Dflydev\FigCookies\SetCookie;
-use Lcobucci\JWT\Builder;
-use Lcobucci\JWT\Parser;
-use Lcobucci\JWT\Signer;
-use Lcobucci\JWT\Token;
-use Lcobucci\JWT\ValidationData;
 use Psr\Http\Message\ResponseInterface as Response;
 use Psr\Http\Message\ServerRequestInterface as Request;
-use PSR7Session\Time\CurrentTimeProviderInterface;
-use PSR7Session\Time\SystemCurrentTime;
-use PSR7Session\Session\DefaultSessionData;
-use PSR7Session\Session\LazySession;
-use PSR7Session\Session\SessionInterface;
 use Zend\Stratigility\MiddlewareInterface;
 
 final class SessionMiddleware implements MiddlewareInterface
```

Comment:
The interface is still implemented, but you removed the dependency, which means that using this class will cause a fatal from now on.